resources/template/index.php

```diff
@@ -0,0 +1,215 @@
+<?php
+$full_images = glob("./{,*/,*/*/,*/*/*/}*.{jpg,JPG}", GLOB_BRACE);
+$tmb_images = glob("./{,*/,*/*/,*/*/*/}tmb_*.{jpg,JPG}", GLOB_BRACE);
+$first_img = $full_images[0];
+$actual_link = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+
+// meta params to evaluate
+$og_locale = 'it_IT';
```

Comment:
Meta params with example text in english instead. Locale should also be en_GB by default.